src/sparseml/pytorch/utils/exporter.py

```diff
@@ -570,7 +570,23 @@ def export_onnx(
 
         # clean up graph from any injected / wrapped operations
         _delete_trivial_onnx_adds(onnx_model)
-    onnx.save(onnx_model, file_path)
```

Comment:
Perhaps we can already to saving to external formats in line L557. There are a couple of things to clarify:
- is there API support (`torch.onnx.export(model, (input, ), './large_model.onnx', use_external_data_format=True)` only available in torch >= 1.15)
- we will need to resave the weights after the "onnx file fixes" L563 anyway, so maybe not worth the trouble.